ui/v2.5/src/components/Tags/TagDetails/TagDetailsPanel.tsx

```diff
@@ -41,9 +43,28 @@ export const TagDetailsPanel: React.FC<ITagDetails> = ({ tag }) => {
           <FormattedMessage id="parent_tags" />
         </dt>
         <dd className="col-9 col-xl-10">
-          {tag.parents.map((p) => (
+          {tag.children.map((p) => (
             <Badge key={p.id} className="tag-item" variant="secondary">
-              <Link to={`/tags/${p.id}`}>{p.name}</Link>
+              <Link to={`/tags/${p.id}`}>
+                {p.name}
+                {p.child_count !== 0 && (
+                  <>
+                    <span className="icon-wrapper">
+                      <span className="vertical-line">|</span>
+                      <OverlayTrigger
+                        placement="top"
+                        overlay={
+                          <Tooltip id="tag-hierarchy-tooltip">
+                            Explore tag hierarchy
+                          </Tooltip>
+                        }
+                      >
```

Comment:
I don't understand why this overlay is here. Clicking on the icon doesn't do anything different than clicking elsewhere on the badge. The text needs to be internationalised as well.